modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -170,6 +170,8 @@ public class InternalTableImpl implements InternalTable {
     /** Placement driver. */
     private final PlacementDriver placementDriver;
 
+    private final ClusterNode localNode;
```

Comment:
Please check my comment bellow. I believe that we should retrieve coordinatorId from txManager on tx begin, thus we will not add given field here.